packages/components/src/font-size-picker/stories/index.tsx

```diff
@@ -8,37 +13,22 @@ import { useState } from '@wordpress/element';
  */
 import FontSizePicker from '../';
 
-export default {
+const meta: ComponentMeta< typeof FontSizePicker > = {
 	title: 'Components/FontSizePicker',
 	component: FontSizePicker,
-	argTypes: {
-		initialValue: { table: { disable: true } }, // hide prop because it's not actually part of FontSizePicker
-		fallbackFontSize: {
-			description:
-				'If no value exists, this prop defines the starting position for the font size picker slider. Only relevant if `withSlider` is `true`.',
-		},
-		size: {
-			control: { type: 'radio' },
-			options: [ 'default', '__unstable-large' ],
-		},
-		withReset: {
-			description:
-				'If `true`, a reset button will be displayed alongside the input field when a custom font size is active. Has no effect when `disableCustomFontSizes` or `withSlider` is `true`.',
-			control: { type: 'boolean' },
-			table: {
-				type: 'boolean',
-				defaultValue: { summary: true },
-			},
-		},
-	},
+	argTypes: {},
```

Comment:
We usually keep components _controlled_ in Storybook

```suggestion
	argTypes: {
		value: { control: { type: null } },
	},
```

Reply:
Ah yeah that makes sense 👍